### PR TITLE
Normalizing console statements, removing window

### PR DESCRIPTION
**Pull request by pamelafox**
### High-level description of change

The main point of this PR is to remove "window" from "window.console" statements, as this moves us towards easier webpack integration. We have some places where we use "console &&" before console, so I normalized to that everywhere. However, I'm happy to remove "console &&" if that's not needed.

### Are there performance implications for this change?

No
### Have you added tests to cover this new/updated code?

No, just ran current tests.

### Risks involved

There's some risk of console && not working in all browsers, but at this point, with modern browsers, that seems less likely.
I checked webapp and we have a few different approaches there.

### Are there any dependencies or blockers for merging this?

No

### How can we verify that this change works?

Use the PJS editor, all should work with no errors, particularly debug()

### CODE REVIEW

File: js/output/pjs/pjs-output.js [hunk 1]
```diff
@@ -437,8 +437,7 @@ window.PJSOutput = Backbone.View.extend({
                     // figure out whether we hit this case.
                     var message = i18n._("Error: %(message)s",
                         {message: errors[errors.length - 1].message});
-                    // TODO(jeresig): Find a better way to show this
-                    this.output.$el.find(".test-errors").text(message).show();
```
Comment (pamelafox) [inline js/output/pjs/pjs-output.js]: test-errors doesn't exist anymore (hasn't in a while). We do display this in a hopper message, I believe, but I figured might as well console.warn() it as well

**Comment by pamelafox** [comment]
Note: VSCode fixed up whitespace issues, so it's easier to view the diff with whitespace-ignore on.

**Comment by pamelafox** [comment]
@kevinbarabash Great, I was hoping that was the case. Much simpler code now. (*Especially if you view diff with ignore-whitespace on)

**Comment by kevinbarabash** [comment]
> (*Especially if you view diff with ignore-whitespace on)

I think I used it that feature once a long time ago and then forgot it existed.  It helps a lot.  I wonder if there's a way to turn it on by default.

**Pull request merged**